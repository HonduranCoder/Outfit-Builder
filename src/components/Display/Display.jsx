export default function Display({ catchphrases }) {
  return (
    <div>
      {catchphrases.map((catchphrase) => (
        <p key={catchphrase}>{catchphrase}</p>
      ))}
    </div>
  );
}
