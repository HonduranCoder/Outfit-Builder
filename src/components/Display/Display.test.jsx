import { render, screen } from '@testing-library/react';
import Display from './Display';

it('should render Display component', () => {
  const { container } = render(
    <Display catchphrases={['cheer', 'cry', 'nod']}></Display>
  );
  expect(container).toMatchSnapshot();
});
