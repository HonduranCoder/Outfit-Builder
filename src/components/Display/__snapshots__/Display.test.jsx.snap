// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Display component 1`] = `
<div>
  <div>
    <p>
      cheer
    </p>
    <p>
      cry
    </p>
    <p>
      nod
    </p>
  </div>
</div>
`;
